CONTRIBUTING.md

```diff
@@ -58,6 +56,8 @@ make pre-commit-install
 
 If you are planning to contribute on documentation, extra dependencies can be installed using `poetry install -E docs`. The detail instruction is included [here](https://github.com/GazzolaLab/MiV-OS/blob/main/docs/README.md).
 
+> Depending on the OS (Mac users), setting up `Poetry` might require adding `export PATH="$HOME/.local/bin:$PATH"` to __.bashrc__ or __.zshrc__. See [here](https://stackoverflow.com/questions/70003829/poetry-installed-but-poetry-command-not-found).
```

Comment:
May be better to link to https://python-poetry.org/docs/#installing-with-the-official-installer -- particularly Step 3 which mentions the directory for each platform (Windows, MacOS, Unix).

We can also remove (Mac users) since this could be a problem on any platform.

Reply:
[d296bef](https://github.com/GazzolaLab/MiV-OS/pull/192/commits/d296bef5f297c987f1fc6c681ae66be67750c643)